app/models/digest_run.rb

```diff
@@ -0,0 +1,37 @@
+class DigestRun < ApplicationRecord
+  validates :starts_at, :ends_at, :date, :range, presence: true
+  before_validation :set_range_dates, on: :create
+  validate :ends_at_is_in_the_past
+
+  enum range: %i(daily weekly)
```

Comment:
We might to define the enum as `{ daily: 0, weekly: 1 }` here.

Reply:
0 and 1 would be the default values assigned anyway wouldn't they? I noticed you defined them explicitly in a PR the other day. Any other reason? I've only done that before where you need values that don't correspond to the array index.

Reply:
I always saw it as a measure to stop someone accidentally adding a new one in between and breaking the meaning of the data in the database.

Reply:
Fair enough. Fixed in 737ccd1